templates/workloads/common/cf/service-base-properties.yml

```
@@ -2,7 +2,7 @@ Cluster:
   Fn::ImportValue:
     !Sub '${AppName}-${EnvName}-ClusterId'
 TaskDefinition: !Ref TaskDefinition
-DesiredCount: !Ref TaskCount
+DesiredCount: !GetAtt DynamicDesiredCountAction.DesiredCount
```

Let's wrap this around `.AutoScaling` so that in case `autoscaling` is not set in the manifest and the Lambda is not created the stack succeeds:
```suggestion
{{- if .Autoscaling }}
DesiredCount: !GetAtt DynamicDesiredCountAction.DesiredCount
{{- else }}
DesiredCount: !Ref TaskCount
{{- end}}
```